homeassistant/components/anova/__init__.py

```diff
@@ -17,15 +24,44 @@
 
 async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Set up Anova from a config entry."""
-    devices = [
+    api = AnovaApi(
+        aiohttp_client.async_get_clientsession(hass),
+        entry.data[CONF_USERNAME],
+        entry.data[CONF_PASSWORD],
+    )
+    try:
+        await api.authenticate()
+        online_devices = await api.get_devices()
+    except InvalidLogin as err:
+        raise ConfigEntryNotReady("Login was incorrect, please try again") from err
```

Comment:
But log the error
```suggestion
        return False
```

In the future `raise ConfigEntryAuthFailed` and implement a reauth flow